test/common/StandaloneUtils.hpp

```diff
@@ -25,7 +25,7 @@
     }                                                           \
 } while(0)
 
-#define MAX_STACK_SIZE 570
+#define MAX_STACK_SIZE 640
```

Comment:
Do we have larger stacksize now?

Reply:
Yes, it came out to be around `584` while I was working on the tests. Added additional buffer (`640`) for future increase (just in case).